lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -522,7 +522,11 @@ def _create_node(
             source_exists = self.host_node.tools[Ls].path_exists(
                 path=node_context.os_disk_base_file_path, sudo=True
             )
-            if not source_exists:
+            if source_exists:
```

Comment:
If you are going to add an unrelated bug fix, at least mention the issue in the PR description.

Reply:
Updated the PR description. This change is infact done as a separate commit.